data/zilliqa/assets/zil1s68pllqj8n5tw4exvayqafhw0l03rp546l3mzw/meta.json

```diff
@@ -0,0 +1,14 @@
+@@ -0,0 +1,15 @@	
+	{
+	"name": "ZiLBOX ",
+	"symbol": "ZBOX",
+	"supply": "1900621",
+	"email": "zilbox@protonmail.com",
+	"links": {
+	"telegram": "https://t.me/ZiLBOX",
+	"twitter": "https://twitter.com/ZilliqaBOX"
+	},
+	"gen": {
+	"logo": "logo.png"
```

Comment:
change your logo file so it is a square

Reply:
what i change in logo file please explain

Reply:
just make it a square, right now its a rectangle

Reply:
![logo](https://user-images.githubusercontent.com/85443803/121632811-e0859000-caab-11eb-972d-12c5dd78ef05.png)

Reply:
Please check now its ok Sir.